examples/lib/defaultHtmlStepUi.js

```diff
@@ -81,27 +81,44 @@ function DefaultHtmlStepUi(_sequencer, options) {
           html += "</select>";
         } else {
           let paramVal = step.options[paramName] || inputDesc.default;
-          html =
-            '<input class="form-control target" type="' +
-            inputDesc.type +
-            '" name="' +
-            paramName +
-            '" value="' +
-            paramVal +
-            '" placeholder ="' +
-            (inputDesc.placeholder || "");
-
-          if (inputDesc.type.toLowerCase() == "range") {
-            html +=
-              '"min="' +
-              inputDesc.min +
-              '"max="' +
-              inputDesc.max +
-              '"step="' +
-              inputDesc.step + '">' + '<span>' + paramVal + '</span>';
+          if (step.name === 'blend' && inputDesc.type === 'input') {
```

Comment:
Instead of checking for step name, how about make it an input type in the info.json file?

Reply:
Can you please explain a bit on this?
Thanks!!

Reply:
In the info.json file, we have all the inputs. Each input has its own type like `text`, `select` etc. Btw, there is not input type called `input` as it is mapped to a valid html input type in the [mapHtmltypes.js](https://github.com/publiclab/image-sequencer/blob/main/examples/lib/mapHtmltypes.js) file(added in a new pr). The only types accesible to the ui itself are `text`, `range`, `select`, `number`, anything else like `input` or `URL` is defaulted to `text`.

So I feel like we could add a new type called `textarea` in the info files and make subsequent changes to the mapHtmltypes.js file.

I hoe this explains everything. Thank you!

Reply:
Thank you so much for this.

Reply:
I think you are using an old verison. Please rebase. This is how it looks in the current main branch

https://github.com/publiclab/image-sequencer/blob/e1a113cde183a620eaa9bb0fd0c37520178eb84a/examples/lib/defaultHtmlStepUi.js#L79